### ISSUE TITLE
A11y_Docs - UX_Tutorial_EnterFocus_Screenreader: Screenreader is announcing unnecessary "Hello word" heading level 1 but visually heading is not present.

### ISSUE THREAD

**Issue by SachinChavhan317** [dotnet/try#993]
### GitHubTags:

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopWeb; #Docs - UX; #P1_Docs - UX_Web_Feb22; #Web; #Narrator; #NVDA; #WCAG1.3.1; #FTP; #STP; #ChromiumEdge; #Chrome;

### Environment Details:
Application Name: Docs - UX
#URL: https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/hello-world 
Windows version: Windows 11
Chromium Edge: Version 99.0.1150.7 (Official build) dev (64-bit)
Screen-Readers: Narrator, NVDA[2021.3.1]

### Repro Steps:

1. Launch the url :https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/hello-world 
2. Tab till "tutorial" section
3. Tab till enter focus button and hit enter
4. Turn on screenreader
5. Observe that Screenreader is announcing unnecessary "Hello word" heading level 1 or not.

### Actual:
Screenreader is announcing unnecessary "Hello word" heading level 1 but visually heading is not present.

NOTE: Issue is not observed in NVDA.

### Expected:
Screenreader should not announce unnecessary "Hello word" heading level 1 if visually heading is not present on the page.

![Presentation2](https://user-images.githubusercontent.com/86241494/162749844-914a6be2-02e0-402b-853a-c2f0c94ab5c1.gif)
